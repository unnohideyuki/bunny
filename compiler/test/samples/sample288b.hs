print' n x = putStrLn $ take n $ show x
  

main = do print (pi :: Double)
          print' 17 (exp 1 :: Double)
          print (log 10 :: Double)
          print (sqrt 2 :: Double)
          print (pi ** 2 :: Double)
          print (logBase 2 10 :: Double)
          print (sin 1 :: Double)
          print (cos 1 :: Double)
          print' 17 (tan 1 :: Double)
          print (asin (-1) :: Double)
          print (acos (-1) :: Double)
          print (atan 1 :: Double)
          print (sinh 1 :: Double)
          print' 16 (cosh 1 :: Double)
          print (tanh 1 :: Double)
          print' 16 (asinh 1 :: Double)
          print (acosh 2 :: Double)
          print' 17 (atanh 0.5 :: Double)

