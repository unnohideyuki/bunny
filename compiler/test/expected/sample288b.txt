3.141592653589793
2.718281828459045
2.302585092994046
1.4142135623730951
9.869604401089358
3.3219280948873626
0.8414709848078965
0.5403023058681398
1.557407724654902
-1.5707963267948966
3.141592653589793
0.7853981633974483
1.1752011936438014
1.54308063481524
0.7615941559557649
0.88137358701954
1.3169578969248166
0.549306144334054
